### ISSUE TITLE
Cannot assign to read only property 'crop'

### ISSUE THREAD

**Issue by d0rsha** [sanity-io/sanity#4026]
**Describe the bug**

Crash in browser when applying hotspot to image inside an custom array block 'object' like the example below: 
```
            title: 'Portable Text',
            name: 'portableText',
            type: 'array',
            of: [
                {
                    type: 'block'
                },
                {
                    type: 'object',
                    /../
                    fields: [
                        {
                            type: 'image',
                            /../
                            options: {
                                hotspot: true
                            }
                        }
                    ],
                }
            ]
```

**To Reproduce**
Crash occurs if 'crop' object is set.

1. Go to sanity studio
2. Remove the object that causes the crash
3. Add a new object with image inside to the text editor 
4. Try to apply hotspot

**Expected behavior**
No crash. 

**Screenshots**
![image](https://user-images.githubusercontent.com/19758313/209640815-4441ad3b-10e9-4ccb-9907-64d09acd4ff8.png)

**Which versions of Sanity are you using?**
@sanity/cli (global)           3.1.2 (up to date)
@sanity/color-input            3.0.1 (up to date)
@sanity/eslint-config-studio   2.0.1 (up to date)
@sanity/production-preview    2.33.2 (up to date)
@sanity/vision                 3.1.2 (up to date)
sanity                         3.1.2 (up to date)

**What operating system are you using?**
Windows 11 / google chrome Version 108.0.5359.125

**Which versions of Node.js / npm are you running?**
8.15.0
v16.17.0

**Additional context**
No output in terminal running 'sanity dev'.

Output in browser console:
```
Error: Cannot assign to read only property 'crop' of object '#<Object>'
TypeError: Cannot assign to read only property 'crop' of object '#<Object>'
    at parseSource (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-WULD5UDA.js?v=55090de1:88:22)
    at urlForImage (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-WULD5UDA.js?v=55090de1:130:21)
    at ImageUrlBuilder2.url (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-WULD5UDA.js?v=55090de1:489:16)
    at BaseImageInput.renderPreview (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-N7XDUJRF.js?v=55090de1:84739:119)
    at http://localhost:3333/node_modules/.sanity/vite/deps/chunk-N7XDUJRF.js?v=55090de1:84871:1149
    at http://localhost:3333/node_modules/.sanity/vite/deps/chunk-N7XDUJRF.js?v=55090de1:81938:59
    at updateMemo (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-D5FNAF4U.js?v=55090de1:16826:27)
    at Object.useMemo (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-D5FNAF4U.js?v=55090de1:17342:24)
    at useMemo (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-WGQJK4T4.js?v=55090de1:1014:29)
    at ObjectField2 (http://localhost:3333/node_modules/.sanity/vite/deps/chunk-N7XDUJRF.js?v=55090de1:81938:52)
```

**Comment by kamerat** [comment]
I have the same exact issue.

**Comment by UK090483** [comment]
Same issue.

**Comment by ninaandal** [comment]
Thank you for reporting @d0rsha 🙌 A fix has been included in [v3.2.5](https://github.com/sanity-io/sanity/releases/tag/v3.2.5)
